netlify/next-runtime
# fix: fix Invalid URL error when using uppercase i18n url

### Summary

When adding a locale such as `de-DE` to next.config.ts, we were giving an Invalid URL error as it our cdn lowercases paths. By adding a rewrite when these locale redirects are happening, we are able to point at the correct site.

### Test plan

1. Visit the middleware  Deploy Preview `/de-DE/static` route
2. Observe that the page shows up without an `Invalid URL` error

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
Fixes https://github.com/netlify/pod-ecosystem-frameworks/issues/300
<img width="912" alt="Screen Shot 2022-11-24 at 2 59 27 PM" src="https://user-images.githubusercontent.com/4700602/203857060-9d5a7d2b-e9cb-47c5-a13f-8a8409e6e184.png">

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
